src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -390,6 +370,31 @@ class UserService(protected val userInfo: UserInfo, dataSource: DataSource, prot
     }
   }
 
+  def overwriteGroupMembersInternal(group: RawlsGroup, users: Set[RawlsUser], subGroups: Set[RawlsGroup], txn: RawlsTransaction): Future[Option[ErrorReport]] = {
```

Comment:
Any changes from the non-internal version?  Looks identical to me but I wanted to check.